src/corehost/corehost.cpp

```diff
@@ -333,6 +337,51 @@ void buffering_trace_writer(const pal::char_t* message)
 {
     g_buffered_errors.append(message).append(_X("\n"));
 }
+
+// Determines if the current module (should be the apphost.exe) is marked as Windows GUI application
+// in case it's not a GUI application (so should be CUI) or in case of any error the function returns false.
+bool get_windows_graphical_user_interface_bit()
+{
+    // The first two bytes of a PE file are a constant signature.
+    const UINT16 PEFileSignature = 0x5A4D;
```

Comment:
You can use constants for these from the standard Windows headers. This one is `IMAGE_DOS_SIGNATURE`